weed/command/export.go

```diff
@@ -62,6 +63,24 @@ var (
 	localLocation, _             = time.LoadLocation("Local")
 )
 
+func printNeedle(vid storage.VolumeId, n *storage.Needle, version storage.Version, deleted bool) {
+	key := storage.NewFileIdFromNeedle(vid, n).String()
+	size := n.DataSize
+	if version == storage.Version1 {
+		size = n.Size
+	}
+	fmt.Printf("%s;%s;%d;%t;%s;%d;%s;%t\n",
```

Comment:
use comma, and maybe escape on name since it may also contain comma.

Or we can just use tsv instead of csv.